libs/net/dal/Migrations/1.0.77/Up/PostUp/00-Content.sql

```diff
@@ -1,8 +1,8 @@
 DO $$
 BEGIN
 
-UPDATE public.content SET summary = '' WHERE summary = '[TBD]';
+UPDATE public.content SET summary = '' WHERE summary IN ('[TBD]', '<p>TBD</p>');
```

Comment:
This won't actually do anything.  We'll need to update TEST manually.

Reply:
Right, or maybe we could add a new migration just for this update?

Reply:
I wouldn't bother.  We can just run this manually.